refactor(editor): use warning background when the plugin is deactived

Sysix opened the pull request:
On: 
![grafik](https://github.com/user-attachments/assets/057eaa57-dcdc-414f-98d5-1ff5e593009e)

Off:
![grafik](https://github.com/user-attachments/assets/bcab0fab-93ff-4984-8f74-3ba2ca8724ba)

Sysix commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10085?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#10085** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10085?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10085?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#10084** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10084?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#10083** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10083?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=10085).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **Mar 28, 12:33 AM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
refactor(editor): use warning background when the plugin is deactived (#10085)

On:
![grafik](https://github.com/user-attachments/assets/057eaa57-dcdc-414f-98d5-1ff5e593009e)

Off:
![grafik](https://github.com/user-attachments/assets/bcab0fab-93ff-4984-8f74-3ba2ca8724ba)
